.palantir/revapi.yml

```diff
@@ -1365,6 +1365,18 @@ acceptedBreaks:
       justification: "Removing deprecations for 1.10.0"
   "1.10.0":
     org.apache.iceberg:iceberg-core:
+      - code: "java.field.constantValueChanged"
```

Comment:
I think this is a case where we can justify breaking the API since the constant was wrong in the first place and nobody should have been using it anyway because the support for it didn't exist

Reply:
Agreed with reasoning, but it would be better if we can also highlight this into justification on why break is acceptable 
> Constant was incorrect and had no functional implementation